packages/compass/src/main/telemetry.ts

```diff
@@ -169,7 +172,6 @@ class CompassTelemetry {
           event: 'Telemetry Disabled',
           properties: {},
         });
-        void this._flushTelemetryAndIgnoreFailure();
```

Comment:
this can't be done anymore without closing the client

Reply:
Just making sure - does this mean that there is a chance that we might lose some telemetry that isn't synced yet? Probably not, right?

Reply:
That should be covered by the other flush, i think the reason this was here before  is to "send telemetry one last time" and then stop sending anything after it has been disabled. I don't remember any strict requirement about this, I think that it should be fine to let the flush happen with a bit of delay here.